platform/broadcom/docker-syncd-brcm/base_image_files/bcmcmd

```diff
@@ -1,3 +1,29 @@
 #!/bin/bash
+function print_error() {
+    echo "No such container $NS"
+    echo "usage bcmcmd <container name> <command>"
```

Comment:
what do you mean by <container name>? 

should we just say bcmcmd -n [asicid] "cmd"?

add an option seems better.

Reply:
Changed to `bcmcmd -n [asicid] "cmd"` in the latest commit